Source/Wrapper/CMakeLists.txt

```diff
@@ -22,24 +22,28 @@ add_dependencies(${DLL_NAME} ShaderConductor)
 
 set_target_properties(${DLL_NAME} PROPERTIES FOLDER "Wrapper")
 
-set(CSHARP_TEST CSharpPinvoke)
+if(SC_WITH_CSHARP)
```

Comment:
Can be:
if(NOT SC_WITH_CSHARP)
	return()
endif()
to avoid indent most lines in this file.